src/Control/Plus.purs

```diff
@@ -2,5 +2,12 @@ module Control.Plus where
 
 import Control.Alt
 
+-- | The `Plus` type class extends the `Alt` type class with a value that
+-- | should be the left and right identity for `(<|>)`.
+-- |
```

Comment:
Perhaps worth mentioning the similarity to `Monoid` as with `Semigroup` for `Alt`?